types/protractor-http-mock/index.d.ts

```diff
@@ -144,31 +146,30 @@ declare namespace mock {
         /**
          * All available request types.
          */
-        type AllRequests<T1, T2> = Get<T1> |
-            PostData<T1, T2> |
-            Post<T1> |
-            Head<T1> |
-            Delete<T1> |
-            Put<T1> |
-            Patch<T1> |
-            Jsonp<T1>;
+        type AllRequests = Get<any> |
+            PostData<any, any> |
+            Post<any> |
+            Head<any> |
+            Delete<any> |
+            Put<any> |
+            Patch<any> |
+            Jsonp<any>;
 
         /**
          * GET request mock.
          */
         interface Get<TResponse> {
             request: {
-                method: Method;
+                method: "GET";
```

Comment:
Just being more specific on which HTTP method should be used in which interface. This is in accordance to documentation put on top of the interfaces.